tests/ui_tests/app/page/cluster/page.py

```diff
@@ -54,35 +57,36 @@ def __init__(self, driver, base_url, cluster_id: int):
         self.table = CommonTableObj(self.driver, self.base_url)
 
     def open_main_tab(self):
-        self.find_and_click(ClusterMenuLocators.main_tab)
+        self.find_and_click(ObjectPageMenuLocators.main_tab)
 
     def open_services_tab(self):
-        self.find_and_click(ClusterMenuLocators.services_tab)
+        self.find_and_click(ObjectPageMenuLocators.services_tab)
 
     def open_hosts_tab(self):
-        self.find_and_click(ClusterMenuLocators.hosts_tab)
+        self.find_and_click(ObjectPageMenuLocators.hosts_tab)
 
     def open_components_tab(self):
-        self.find_and_click(ClusterMenuLocators.components_tab)
+        self.find_and_click(ObjectPageMenuLocators.components_tab)
 
     def open_config_tab(self):
-        self.find_and_click(ClusterMenuLocators.config_tab)
+        self.find_and_click(ObjectPageMenuLocators.config_tab)
 
     def open_status_tab(self):
-        self.find_and_click(ClusterMenuLocators.status_tab)
+        self.find_and_click(ObjectPageMenuLocators.status_tab)
 
     def open_import_tab(self):
-        self.find_and_click(ClusterMenuLocators.import_tab)
+        self.find_and_click(ObjectPageMenuLocators.import_tab)
 
     def open_actions_tab(self):
-        self.find_and_click(ClusterMenuLocators.actions_tab)
+        self.find_and_click(ObjectPageMenuLocators.actions_tab)
 
 
 class ClusterMainPage(ClusterPageMixin):
     """Cluster page Main menu"""
 
     MENU_SUFFIX = 'main'
 
+    @allure.step("Check main elements on page")
     def check_all_elements(self):
```

Comment:
I suggest name allure step like "Check all main elements on the page are presented", so we know what exactly is checked
And there is another method with the same name, so maybe it's better to move it to Mixin and add an attribute like `MAIN_ELEMENTS` or `BASE_ELEMENTS`, so we can just override it in descendants

Reply:
good idea! changed